test/api/v3/integration/dataexport/GET-export_userdata.xml.test.js

```diff
@@ -8,7 +8,7 @@ let parseStringAsync = Bluebird.promisify(xml2js.parseString, {context: xml2js})
 
 describe('GET /export/userdata.xml', () => {
   // TODO disabled because it randomly causes the build to fail
```

Comment:
can you remove this comment?

Reply:
Yep, missed this in my first pass. Removed.